paramgen/internal/paramgen_test.go

```diff
@@ -37,14 +37,14 @@ func TestParseSpecificationSuccess(t *testing.T) {
 			want: map[string]config.Parameter{
 				"foo": {
 					Default:     "bar",
-					Description: "foo is a required field in the global config with the name \"foo\" and default value \"bar\".",
+					Description: "MyGlobalString is a required field in the global config with the name\n\"foo\" and default value \"bar\".",
```

Comment:
Yeah, that was the initial idea. It doesn't always work out nicely though. For example, I noticed that we have configuration fields that are named `Field`, it might be unexpected to then change `Field` to `foo` everywhere, since it's a word that might not refer to the name of the field. Same with words like `URL` or `Secret` or similar. The other reason is that the sentence will start with a lowercase letter, which looks strange. I'd rather that we just write the comment as we want it to appear in the docs.

The only issue is that the linter will warn us if we don't start the sentence with the name of the field, but we shouldn't let the linter make our lives hard, we can add an exception if that actually happens.